site/content/en/docs/Installation/Creating Cluster/aks.md

```diff
@@ -0,0 +1,95 @@
+---
+title: "Azure Kubernetes Service"
+linkTitle: "Azure"
+weight: 30
+description: >
+  Follow these steps to create a cluster and install Agones directly on [Azure Kubernetes Service (AKS) ](https://docs.microsoft.com/azure/aks/).
+---
+
+## Choosing your shell
+
+You can use either [Azure Cloud Shell](https://docs.microsoft.com/azure/cloud-shell/overview) or install the [Azure CLI](https://docs.microsoft.com/cli/azure/?view=azure-cli-latest) on your local shell in order to install AKS in your own Azure subscription. Cloud Shell comes preinstalled with `az` and `kubectl` utilities whereas you need to install them locally if you want to use your local shell. If you use Windows 10, you can use the [WIndows Subsystem for Windows](https://docs.microsoft.com/windows/wsl/install-win10) as well.
+
+## Creating the AKS cluster
+
+If you are using Azure CLI from your local shell, you need to login to your Azure account by executing the `az login` command and following the login procedure.
+
+Here are the steps you need to follow to create a new AKS cluster (additional instructions and clarifications are listed [here](https://docs.microsoft.com/azure/aks/kubernetes-walkthrough)):
+
+{{% feature expiryVersion="1.2.0" %}}
+```bash
+# Declare necessary variables, modify them according to your needs
+AKS_RESOURCE_GROUP=akstestrg     # Name of the resource group your AKS cluster will be created in
+AKS_NAME=akstest     # Name of your AKS cluster
```

Comment:
nit: it'd be nice to add some space and line up the end of line comment blocks vertically.